cli/root_internal_test.go

```diff
@@ -67,7 +67,11 @@ func Test_formatExamples(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
-	// Replace with goleak.VerifyTestMain(m) when we enable goleak.
-	os.Exit(m.Run())
-	// goleak.VerifyTestMain(m)
+	goleak.VerifyTestMain(m,
+		// The lumberjack library is used by by agent and seems to leave
+		// goroutines after Close(), fails TestGitSSH tests.
+		// https://github.com/natefinch/lumberjack/pull/100
```

Comment:
Should we do a go mod replace instead?

Reply:
TBH I haven't done a proper review of that PR, and I'm not confident in using it in a production environment without doing so. My judgement was that this is a non-issue in a production environment since the agent only runs once and thus won't be leaking memory, so I think the goleak ignore is acceptable here.

Reply:
Makes sense! TY sir